### PR TITLE
Fix: cohorts get stuck when passing strings instead of numbers.

### PR DESCRIPTION
**Pull request by arikfr**
Parse the value to get a number.

### CODE REVIEW

File: client/app/visualizations/cohort/index.js [hunk 1]
```diff
@@ -115,17 +115,21 @@ function prepareSimpleData(sortedData, options) {
 function prepareData(rawData, options) {
   rawData = _.map(rawData, item => ({
     date: item[options.dateColumn],
-    stage: item[options.stageColumn],
-    total: item[options.totalColumn],
-    value: item[options.valueColumn],
+    stage: parseFloat(item[options.stageColumn]),
```
Comment (kravets-levko) [inline client/app/visualizations/cohort/index.js]: `stage` probably can be even `integer`

Comment (arikfr) [reply client/app/visualizations/cohort/index.js]: https://github.com/getredash/redash/pull/3397/commits/655ce506c82b3165985c1e45b8f4fc3ce45b4c10

File: client/app/visualizations/cohort/index.js [hunk 2]
```diff
@@ -115,17 +115,21 @@ function prepareSimpleData(sortedData, options) {
...
   rawData = _.map(rawData, item => ({
     date: item[options.dateColumn],
-    stage: item[options.stageColumn],
-    total: item[options.totalColumn],
-    value: item[options.valueColumn],
+    stage: parseInt(item[options.stageColumn], 10),
+    total: parseFloat(item[options.totalColumn]),
+    value: parseFloat(item[options.valueColumn]),
   }));
   const sortedData = _.sortBy(rawData, r => r.date + parseInt(r.stage, 10));
```
Comment (kravets-levko) [inline client/app/visualizations/cohort/index.js]: `parseInt` not needed here (value already converted)

Comment (arikfr) [reply client/app/visualizations/cohort/index.js]: 👍 removed.

**Pull request merged**